scripts/src/main/resources/scripts/command/gcviewer

```diff
@@ -0,0 +1,84 @@
+#!/usr/bin/env bash
+
+# autocompletion list
+if [ "${1}" = "shortlist" ]
+then
+  if [ -z "${2}" ]
+  then
+    echo "setup start help"
+  fi
+  exit
+fi
+
+if [ -n "${DEVON_IDE_TRACE}" ]; then set -vx; fi
+# shellcheck source=scripts/functions
+source "$(dirname "${0}")"/../functions
+
+GCVIEWER_HOME="${DEVON_IDE_HOME}/software/gcviewer"
+
+# gcviewer installation
+function doSetup(){
+
+  if [ -z "${GCVIEWER_VERSION}" ]
```

Comment:
silent option should be supported by all commandlets.
Can you check any other commandlet and incorporate these changes here?